src/styles/blocks/button.scss

```diff
@@ -0,0 +1,11 @@
+@use '../_variables/variables' as *;
+
+.button {
+  border: none;
+  border-radius: 5px;
+  padding: 12px 0;
+  text-align: center;
+  font-size: 14px;
+  font-weight: $font-bold;
```

Comment:
The variable `$font-bold` should be defined in the `variables.scss` file. Please ensure that it is correctly defined and imported.